core/src/main/java/hudson/model/BooleanParameterDefinition.java

```diff
@@ -30,19 +30,28 @@
 import hudson.Extension;
 
 import java.util.Objects;
+import org.kohsuke.stapler.DataBoundSetter;
 
 /**
  * {@link ParameterDefinition} that is either 'true' or 'false'.
  *
  * @author huybrechts
  */
 public class BooleanParameterDefinition extends SimpleParameterDefinition {
-    private final boolean defaultValue;
+    private boolean defaultValue;
 
+    /**
+     * @since XXX
```

Comment:
```suggestion
     * @since TODO
```

Plus elsewhere, for compatibility with https://github.com/jenkinsci/jenkins/blob/71674c92c9118ff7cd4d34f508e7da9b2571399d/update-since-todo.sh#L13

Reply:
OK, I could not remember which it was!